debian/changelog

```diff
@@ -1,10 +1,13 @@
 cloud-init (24.2-0ubuntu1~24.04.2) UNRELEASED; urgency=medium
 
-  * Upstream snapshot based on upstream/main at b7b11bc0.
   * d/p/no-single-process.patch: Remove single process optimization
   * d/p/no-nocloud-network.patch: Remove nocloud network feature
+  * refresh patches:
+    - d/p/no-nocloud-network.patch
+    - d/p/no-single-process.patch
```

Comment:
Let's drop this automatically added content, no need to represent refresh patches on patches introduced and yet to be released.